provider/pkg/gen/examples/upstream/ingress.md

```diff
@@ -162,6 +162,29 @@ func main() {
 	})
 }
 ```
+```yaml
+description: Create an Ingress with auto-naming
+name: yaml-example
+resources:
+    ingress:
```

Comment:
It might be helpful to name the resources something different than the type to make the usage clearer.